src/pybind/mgr/pg_autoscaler/tests/test_autoscaler.py

```diff
@@ -1,34 +0,0 @@
-from pg_autoscaler import effective_target_ratio
```

Comment:
looks like this file got removed by accident

Reply:
still appearing deleted in this commit

Reply:
Yea sorry I decided to rename the files so it is less complicated, I forgot to commit the new file. However, this should be fixed now